src/js/apps/patients/worklist/worklist_sort.js

```diff
@@ -33,10 +33,10 @@ const SortOption = Backbone.Model.extend({
       const bValue = getEntityFieldValue(b.model, fieldName, keys);
 
       if (sortType === 'numeric') {
-        return numSort(direction, aValue ?? 0, bValue ?? 0);
+        return numSort(direction, aValue, bValue);
       }
 
-      return alphaSort(direction, aValue ?? '', bValue ?? '');
+      return alphaSort(direction, aValue, bValue);
```

Comment:
`alphaSort` already converts null values to `''` by default (via [this code](https://github.com/RoundingWell/care-ops-frontend/blob/develop/src/js/utils/sorting.js#L8)). So the `??` check isn't needed. Which was causing the coveralls missing branch issue.

Also updated `numSort()` to convert null values to `0`. So we can remove the `??` checks for that too.